Automattic/wp-calypso
# Add multi-year domain purchases

This PR enables users to purchase domains for more than 1 year. As this is a big change, we're adding a feature flag to enable it when we're ready.


### Screenshot

![image](https://cloud.githubusercontent.com/assets/991022/9464907/5f410020-4b30-11e5-9255-052b8c989a84.png)


### Testing

You'll need to test this with D341-code.

- Add a domain to the cart
- Go to checkout
- Change the year from the select box
- Buy the domain

- Make sure the domain is registered for the duration you have picked
- Make sure you are charged the correct amount
- If you opted for a privacy protection, make sure you are charged for it correctly as well.

- Cancel the domain you've just registered
